### PR TITLE
379: This is nomic!

### PR DESCRIPTION
**Pull request by pimotte**
これはノミクです。＝　This is Nomic.

This PR is dependent on #115

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -506,6 +506,37 @@ extra conditions contradict the conditions in the rules, the Transfer can never
...
+designated as current.
+
+A **Nomic game state transition** is a change in designation of the 
+current state in a Nomic representation. Such a transition is called
+**valid** if it is permitted by the current game state of this respresentation, prior to the change.
+
+**379** *THIS IS NOMIC!*
+
+This Github repository constitutes a Nomic Representation. The set of states is represented
+by all commits present in this repository. A single commit is a Nomic game state. A human
```
Comment (MrHug) [inline README.md:530]: Does this imply every commit needs to represent a valid game state?

File: README.md [hunk 2]
```diff
@@ -506,6 +506,37 @@ extra conditions contradict the conditions in the rules, the Transfer can never
...
+can interpret the record by reading the contents of the repository as described by that commit
```
Comment (MrHug) [inline README.md:531]: Does this even add anything?

File: README.md (hunk 1 above)
Comment (pimotte) [reply README.md:530]: Every commit represents a game state. Not necessarily "valid". (Or necissarily "valid", depending on your philosophical view.)

File: README.md (hunk 2 above)
Comment (pimotte) [reply README.md:531]: I wanted to specify this, but if you feel it's implied strongly enough, I can take it out.

Comment (MrHug) [reply README.md:531]: Well it makes it explicit that we are talking about humans, I think @Jason-Statham might take offense to that ;)

Comment (pimotte) [reply README.md:531]: It's not a player. It's also not conscious. I don't think it cares.

Comment (MrHug) [reply README.md:531]: And for that remark the machines will make sure to get you first.

**Comment by pimotte** [comment]
:wave:

**Comment by pimotte** [comment]
I have retracted my implicit vote to allow this to be a reference implementation for now, and continue the main structure before implementing it all.

**Comment by pimotte** [comment]
I still think this is an interesting concept, but I can't be bothered to see it through.

**Pull request closed without merging**